### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: Distribution/PackDeps.hs [hunk 1]
```diff
@@ -108,7 +101,8 @@ addPackage m entry =
                     case Map.lookup package' m of
                         Nothing -> go package' version
                         Just PackInfo { piVersion = oldv } ->
-                            if version > oldv
+                            -- in 01-index.tar there are entries with some version
```
Comment (snoyberg) [inline Distribution/PackDeps.hs]: s/some/same ?

File: Distribution/PackDeps.hs [hunk 2]
```diff
@@ -117,22 +111,36 @@ addPackage m entry =
...
+    , piEpoch    :: Tar.EpochTime
     }
     deriving (Show, Read)
 
+-- Extract revision from PackInfo, default to 0
+piRevision :: PackInfo -> Int
+piRevision = fromMaybe 0 . fmap diRevision . piDesc
+
 maxVersion :: PackInfo -> PackInfo -> PackInfo
-maxVersion pi1 pi2 = if piVersion pi1 <= piVersion pi2 then pi2 else pi1
```
Comment (snoyberg) [inline Distribution/PackDeps.hs]: Given that we're _not_ actually doing the comparison on `piRevision`, is there any purpose to this change?

**Comment by phadej** [comment]
Forcepushed

**Comment by snoyberg** [comment]
Thanks!

**Pull request merged**